spec/integration/api/users_controller_spec.rb

```diff
@@ -0,0 +1,612 @@
+# frozen_sTring_literal: true
+
+require "spec_helper"
+
+describe Api::V1::UsersController do
+  include PeopleHelper
+
+  let(:full_scopes) {
+    %w[openid public:read public:modify private:read private:modify contacts:read profile profile:modify]
+  }
+
+  let(:auth) {
+    FactoryGirl.create(
+      :auth_with_default_scopes,
+      scopes: full_scopes,
+      user: FactoryGirl.create(:user, profile: FactoryGirl.create(:profile_with_image_url))
```

Comment:
Layout/AlignHash: Align the elements of a hash literal if they span more than one line.